scripts/tasks.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+# ./script/task.sh: Run a VSCode task from tasks.json including its dependencies.
```

Comment:
this is evil and i love it

id also like if you could copy in the nix trick from the other script but it's not required

Reply:
Sure, still no clue what `nix` is, but I bet it's lovely